Source/WebKit/_WebKit_SwiftUI/View+WebViewModifiers.swift

```diff
@@ -48,7 +48,7 @@ extension EnvironmentValues {
     var webViewContentBackground: Visibility = .automatic
 
     @Entry
-    var webViewOnScrollGeometryChange: OnScrollGeometryChangeContext? = nil
+    var webViewOnScrollGeometryChange = OnScrollGeometryChangeContext()
```

Comment:
Instead of modifying `AnyEquatable` I bet we could sink that special case into `OnScrollGeometryChangeContext` itself. Perhaps even keeping some flag or `enum` there.